### ISSUE TITLE
improve install scenario speed

### ISSUE THREAD

**Issue by behrisch** [DLR-TS/tsc#50]
current stats for berlin_net:

- net building: 3 minutes
- taxi stops: 4 minutes
- bidi taz: 22 minutes
- gtfs2pt: 89 minutes
- district mapping: 3 minutes per shape input
- suburb mapping: 18 minutes

**Comment by namdre** [comment]
computeBidiDistricts.py might be made to run faster if `getCandidates` where to do vClass filtering. I imagine there are a lot of useless pedestrian paths within a 500m radius around any given edge that have an quadratic impact due to `symmetrical=True`.

**Comment by behrisch** [comment]
yes and no, the bidi taz are used for pedestrians as well but we might want to change that
